odoo/addons/base/transpiler/sourcemap_generator.py

```diff
@@ -0,0 +1,187 @@
+from functools import lru_cache
+
+
+class SourceMapGenerator:
+    """
+    The SourceMapGenerator will create the sourcemap that maps the js files to the asset bundle.
```

Comment:
will create -> creates (in general, write in present, it's much more easier to read)

Reply:
"that maps the js files to the asset bundle" I would state this the other way around: "maps the asset bundle to the js files"